handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -53,6 +57,15 @@
 public class LoadbalanceHandler extends AbstractHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(LoadbalanceHandler.class);
 
+    private static final ExecutorService RETRY_POOL = Executors.newCachedThreadPool(new ThreadFactory() {
```

Comment:
This Is not a resource leak stuff, but a performance stuff I think. And I.think it's better to use a cached pool to reuse in high volume request and when no requests thread will be destroyed  automatically